_includes/header.html

```diff
@@ -29,6 +29,7 @@
         <li><a href="/adopt">Adopt</a></li>
         <li><a href="/join">Join</a></li>
         <li><a href="/events">Events</a></li>
+        <li><a href="http://shop.gpa-centex.com/">Shop</a></li>
```

Comment:
I believe Jorden is planning to implement HTTPS for the shop, which means this URL will need to change from `http` to `https`

Reply:
Also, I believe we are going to change the URL from `.com` to `.org` to match the main website.